apps/site/lib/site/content_rewriter.ex

```diff
@@ -33,6 +33,17 @@ defmodule Site.ContentRewriter do
     rewrite(Enum.join(content), conn)
   end
 
+  @doc """
+  Certain CMS content should have its HTML content passed through (scripts, iframes, etc.).
+  This content is not scrubbed, santitized, or rewritten due to its variability and complexity.
+  """
+  @spec rewrite_raw(Phoenix.HTML.safe() | String.t(), Plug.Conn.t()) :: Phoenix.HTML.safe()
+  def rewrite_raw(content, _conn) do
```

Comment:
Could you please add some tests for this function?

Reply:
I found this function to be unnecessary, so removed it. See recent code changes.